index.html

```diff
@@ -0,0 +1,483 @@
+          />
+          <h3 class="home-feature__card-title">
+            Customizable AI<br />
+            features
+          </h3>
+          <p class="home-feature__card-description">
+            Lorem ipsum dolor sit amet, consetetur<br />
+            sadipscing elitr, sed diam nonumy eirmod<br />
+            tempor.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="home-testimonials">
+      <h1 class="home-testimonials__title">
+        The stunning results our<br />
+        customers have experienced
+      </h1>
+      <div class="home-testimonials__card-container">
+        <div class="home-testimonials__card">
+          <p class="home-testimonials__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="ome-testimonials__card-profile">
+            <img src="./assets/images/person1.png" alt="photo-1" />
+            <p class="ome-testimonials__card-profile-name">Ron Wood</p>
+            <p class="ome-testimonials__card-profile-role">CEO</p>
+          </div>
+        </div>
+        <div class="home-testimonials__card">
+          <p class="home-testimonials__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="ome-testimonials__card-profile">
+            <img src="./assets/images/person2.png" alt="photo-1" />
+            <p class="ome-testimonials__card-profile-name">Mark Mason</p>
+            <p class="ome-testimonials__card-profile-role">Marketing Manager</p>
+          </div>
+        </div>
+        <div class="home-testimonials__card">
+          <p class="home-testimonials__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua.
+          </p>
+          <div class="ome-testimonials__card-profile">
+            <img src="./assets/images/person3.png" alt="photo-1" />
+            <p class="ome-testimonials__card-profile-name">Sam Preston</p>
+            <p class="ome-testimonials__card-profile-role">CTO</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- footer container -->
+    <section class="footer-container">
+      <div class="footer-container__header">
+        <h1 class="footer-container__title">
+          Are you ready to grow your<br />
+          business with us?
+        </h1>
+        <p>
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+        <button class="primary-btn">View Pricing</button>
```

Comment:
## prohibition of nested block
`primary-btn` you are using block inside an other block is not a best practice